EventFilter/CSCRawToDigi/src/CSCDDUEventData.cc

```diff
@@ -193,6 +193,14 @@ void CSCDDUEventData::unpack_data(const uint16_t* buf, CSCDCCExaminer* examiner)
         if (cscid != -1) {
           const uint16_t* pos = (const uint16_t*)csc_itr->second;
 
+          if (pos == nullptr) {
+            if (debug)
+              LogTrace("CSCDDUEventData|CSCRawToDigi")
+                  << "skip unpacking of CSC " << cscid << " due format errors (NULL pointer to chamber data)"
+                  << std::dec;
```

Comment:
```suggestion
                  << "skip unpacking of CSC " << cscid << " due to format errors (NULL pointer to chamber data)"
                  << std::dec;
```
Just a small typo. One question: is the idea to print `cscid` as "hex", and then switch to `std::dec` ?

Reply:
@missirol thanks! (cut-n-paste legacy typos from other log message few lines below). Fixed both.